lib/cli/kit/config.rb

```diff
@@ -29,11 +29,13 @@ def get(section, name, default: nil)
 
       # Coalesce and enforce the value of a config to a boolean
       def get_bool(section, name, default: false)
-        case get(section, name, default: default).to_s
+        case get(section, name, default: default)
         when 'true'
           true
         when 'false'
           false
+        when default
+          default
```

Comment:
Should we restrict the defaults to `true`, `false`, or `nil`, or do we just trust that nobody would pass a non (nilable) boolean as default to a method called `get_bool`?

Reply:
I mean, if someone wants to pass `:unset`, I wouldn't object.